javascript/ql/lib/change-notes/2025-03-26-async-fileRead.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Added data flow from `require("fs").promises.readFile()` to `async`.
```

Comment:
This message is unclear. I don't see the connection between `requre..readFile()` and `async`.  

How about this?

```suggestion
* Improved modeling of the `node:fs` module: `await`-ed calls to `read` and `readFile` are now supported.
```